integration_tests/tests/cucumber.rs

```diff
@@ -3857,6 +3795,104 @@ async fn node_reached_sync(world: &mut TariWorld, node: String) {
     )
 }
 
+#[when(expr = "I create a burn transaction of {int} uT from {word} at fee {int}")]
+async fn burn_transaction(world: &mut TariWorld, amount: u64, wallet: String, fee: u64) {
+    let mut client = world.get_wallet_client(&wallet).await.unwrap();
+
+    let req = grpc::CreateBurnTransactionRequest {
+        amount,
+        fee_per_gram: fee,
+        message: "Burning some tari".to_string(),
+    };
+
+    let result = client.create_burn_transaction(req).await.unwrap();
+    let tx_id = result.into_inner().transaction_id;
+
+    let mut last_status = 0;
+    for _ in 0..(TWO_MINUTES_WITH_HALF_SECOND_SLEEP) {
+        let result = client
+            .get_transaction_info(grpc::GetTransactionInfoRequest {
+                transaction_ids: vec![tx_id],
+            })
+            .await
+            .unwrap();
+
+        last_status = result.into_inner().transactions.last().unwrap().status;
+
+        if let 1 | 2 | 6 = last_status {
```

Comment:
Can't we use enum values instead of 1,2,6?

Reply:
We can, it just seemed faster. I'll put the enum in.